crates/uv-resolver/src/manifest.rs

```diff
@@ -100,39 +100,40 @@ impl Manifest {
     ) -> impl Iterator<Item = &Requirement> + 'a {
         match mode {
             // Include all direct and transitive requirements, with constraints and overrides applied.
-            DependencyMode::Transitive => Either::Left( self
```

Comment:
No changes here, i just nudged rustfmt into right direction (i think it didn't like the comma placement)